DigitalLibraryofTennessee/DLTN_XSLT
# Chatt pub

**GitHub Issue**: [295](https://github.com/DigitalLibraryofTennessee/DLTN_XSLT/issues/295)

* Other Relevant Links (OAI Feeds, DPLA Records, Metadata Mappings, DLTN Documentation, Etc.)
* https://collections.chattlibrary.org/oai?verb=ListRecords&metadataPrefix=oai_dcterms&set=localhistory

## What does this Pull Request do?

This PR adds a transform for Chattanooga Public Library. They are adding content to DPLA for the first time and use Omeka.

## How should this be tested?

Check the resulting output of the transform to see if it accounts for the required DPLA elements and generates valid MODS. Here's some sample XML
[chattpub_localhistory.txt](https://github.com/DigitalLibraryofTennessee/DLTN_XSLT/files/11843313/chattpub_localhistory.txt). Particularly take a look at the preview and object in context links to make sure that they resolve. Note that Chattanooga Public Library doesn't seem to have thumbnails for PDF objects. The thumbnail URL for jpg items was also altered so that it includes "medium" instead of "original."

## Interested parties

@markpbaggett
